docs/src/pages/css-in-js/basics/basics.md

```diff
@@ -10,13 +10,13 @@ You can use it, but you don't have to. This styling solution is [interoperable w
 
 In previous versions, Material-UI has used LESS, then a custom inline-style solution to write the style of the
 components, but these approaches have proven to be limited. Most recently, we have [moved toward](https://github.com/oliviertassinari/a-journey-toward-better-style)
-a *CSS-in-JS* solution. It **unlocks many great features** (theme nesting, dynamic styles, self-support, etc.).
+a *CSS in JS* solution. It **unlocks many great features** (theme nesting, dynamic styles, self-support, etc.).
```

Comment:
Why changing? The hyphenated version seems to be widely used.

Reply:
We were using the two variants. Try a Google search, everybody is contradicting itself. Ok, let's trust this variant: https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660.
There are relatively few results when Google CSS-in-JS, what do you think of renaming it to simple "styling"?

Reply:
https://next.material-ui.com/css-in-js/basics/ would become https://next.material-ui.com/styles/basics/

Reply:
as for https://next.material-ui.com/style/icons/, we can either rename it content, or move it to the components subsection.